packages/deployment-service-ui/src/components/pages/deployments/list.tsx

```diff
@@ -0,0 +1,101 @@
+import Routes from '@/constants/routes'
+import { reapitConnectBrowserSession } from '@/core/connect-session'
+import { ReapitConnectSession, useReapitConnect } from '@reapit/connect-session'
+import { ButtonGroup, H3, Section } from '@reapit/elements'
+import { Button, Table } from '@reapit/elements'
+import { Loader } from '@reapit/elements/v3'
+import { DeploymentModelInterface } from '@reapit/foundations-ts-definitions'
+import React, { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import {
+  deploymentServiceDelete,
+  deploymentServicePaginate,
+  deploymentServiceRun,
+} from '../../../platform-api/deployments'
+
+export default () => {
+  const { connectSession } = useReapitConnect(reapitConnectBrowserSession)
+  const [deployments, setDeployments] = useState<DeploymentModelInterface[]>([])
+  const [loading, setLoading] = useState<boolean>(false)
+  const [deletionLoading, setDeletionLoading] = useState<string[]>([])
+  const [deploying, setDeploying] = useState<string[]>([])
+
+  useEffect(() => {
+    const fetchDeployments = async () => {
+      setLoading(true)
+      const serviceResponse = await deploymentServicePaginate(connectSession as ReapitConnectSession)
+      setLoading(false)
+      if (serviceResponse) {
+        setDeployments([...deployments, ...serviceResponse])
+      }
+    }
+    if (connectSession) {
+      fetchDeployments()
+    }
+  }, [connectSession])
+
+  const deleteDeployment = async (id: string) => {
```

Comment:
As per above, would break these out into handlers but don't worry about it for this app